pom.xml

```diff
@@ -2511,8 +2511,11 @@
 
     <profile>
       <id>hadoop-2.7</id>
+      <activation>
```

Comment:
IIRC as soon as you set any profile at all, the active by default ones are disabled. This could be problematic.